workflow/scripts/additional_functionality.py

```diff
@@ -41,16 +41,23 @@ def add_capacity_limits(n, investment_year, limits_capacity, sense="maximum"):
                 extendable_index = c.df.index[
                     valid_components & c.df[attr + "_nom_extendable"]
                 ]
+                efficiency = (
+                    c.df.loc[existing_index, "efficiency"].mean()
```

Comment:
No need to use `mean`, better multiply the vectors. 

[For FT there should be no problems, but this function is more general and e.g. for gas turbines there would be a difference]